src/cron/dailyDigest.ts

```diff
@@ -0,0 +1,92 @@
+import fastq from 'fastq';
+import {
+  createEmailBatchId,
+  digestPreferredHourOffset,
+  notifyGeneratePersonalizedDigest,
+} from '../common';
+import {
+  UserPersonalizedDigest,
+  UserPersonalizedDigestSendType,
+} from '../entity';
+import { Cron } from './cron';
+
+const cron: Cron = {
+  name: 'daily-digest',
+  handler: async (con, logger) => {
```

Comment:
Just a suggestion, we can create a generic handler for this and the other cron. They do the same thing except the where clause of the query

Reply:
It is also timestamp calculations and sendType. I'll take a look though, it is duplicate mostly.

Reply:
Added in dd70f0c9824f05e418994f47beedcc285aae8ee6